lib/routes/home/widgets/payments_list/widgets/payment_details_sheet/payment_details_sheet.dart

```diff
@@ -122,11 +128,14 @@ class PaymentDetailsSheet extends StatelessWidget {
                         paymentData: paymentData,
                         labelAutoSizeGroup: _labelGroup,
                       ),
-                      if (bolt11.isNotEmpty) ...<Widget>[
-                        PaymentDetailsSheetBolt11(bolt11: bolt11),
+                      if (invoice != null && invoice.isNotEmpty) ...<Widget>[
+                        PaymentDetailsSheetInvoice(invoice: invoice),
                       ],
                       if (paymentPreimage.isNotEmpty) ...<Widget>[
-                        PaymentDetailsSheetPreimage(paymentPreimage: paymentPreimage),
+                        PaymentDetailsSheetPreimage(
+                          invoice: bolt11 ?? bolt12Offer,
```

Comment:
The bolt12 offer is not the invoice, so I don't see that it will be validated properly

Reply:
You are right. We need to persist Bolt 12 invoice on payment data. Opened an issue on SDK:
- https://github.com/breez/breez-sdk-liquid/issues/646